dev-env/install-ansible.sh

```diff
@@ -13,4 +13,6 @@ dnf install -q -y python3
 python3 -m pip install -r /vagrant/requirements.txt
 
 # Add host keys to `known_hosts`
-ssh-keyscan head >> ~/.ssh/known_hosts
\ No newline at end of file
+mkdir -p ~/.ssh/
+touch ~/.ssh/known_hosts
```

Comment:
I'm surprised you need the `touch` command. Doesn't `>>` append or create the target file? It does when I experiment with bash on CentOS.

Reply:
Oh, this might be an incremental thing (first I thought the file didn't exist, then I figured the directory didn't exist). I'll try removing it in a future commit.